ConversionTool/HtmlToImage/WkHtmlToImage.cs

```diff
@@ -11,10 +9,12 @@ public Task<byte[]> FromStringToPngAsync(string html)
             var imageConfiguration = new ImageConfiguration
             {
                 Content = html,
-                Format = ImageFormat.Png,
+                Format = HtmlConverter.Options.ImageFormat.Png,
+                Quality = 50,
```

Comment:
Se podrían utilizar los siguientes valores de configuración para hacer el resize ?
```suggestion
                Quality = 50,
                Width = xx,
                Height = xx,
```